pyproject.toml

```diff
@@ -1,43 +1,54 @@
+[build-system]
+build-backend = "setuptools.build_meta"
```

Comment:
Is this mandatory? :)

Reply:
If I understand correctly, `setuptools` is the default at the moment
https://peps.python.org/pep-0518/#build-system-table
But I guess it's good to be explicit?

Reply:
OK, then :)